library/control/src/modules/WorkflowManager.rb

```diff
@@ -1231,6 +1231,8 @@ def WorkflowRequiresRegistration(src_id)
     end
 
     def IncorporateControlFileOptions(filename)
+      return false if filename.nil?
```

Comment:
1. how do we know that this is the guilty caller?
2. since we're touching the method, let's document its API and side effects

Reply:
1. Backtrace of the raised exception
2. I'll try that...

Reply:
>     1. Backtrace of the raised exception

Duh. You're right. Now I looked too and found that the `nil` comes from `GetCachedWorkflowFilename` which is even documented.

>     2. I'll try that...

thanks!